src/main/java/seedu/duke/constants/CommandConstants.java

```diff
@@ -10,4 +10,7 @@
     public static final String COMMAND_FOOD_CLEAR = "food clear";
     public static final String COMMAND_FOOD_LIST = "food list";
     public static final String COMMAND_ADD_NOTE = "add";
+    public static final String COMMAND_HELP = "help";
+    public static final String COMMAND_MODULE_SHOW = "module";
```

Comment:
```suggestion
    public static final String COMMAND_MODULE_LIST = "module list";
```

may be more in line with other features of displaying items